playbook.md

```diff
@@ -820,6 +820,44 @@ Designers at dxw follow these principles:
 
    There is always room for improvement. Design can always be made better. There is a difference between ‘done’ and ‘finished’.
 
+### Commercial Operations
+
+We are the team that enables and encourages dxw to achieve its mission(s) and uphold its values.
+
+#### Commercial Operations Team Principles
+
+### Things we do:
```

Comment:
Another structure-related comment: team names are level 3 headings (`###`), the "principles" heading is level 4, so I think "Things we do" and "Behaviours" should be level 5. (They're subordinate to the principles.)